src/builder/create_application_command.rs

```diff
@@ -273,6 +273,24 @@ impl CreateApplicationCommandOption {
 
         self
     }
+
+    /// Sets the minimum permitted length for this string option.
+    ///
+    /// The value of `min_length` must be greater or equal to `0`.
```

Comment:
Isn't this a bit redundant? Unsigned types in Rust cannot go below zero no matter what